lib/beaker/dsl/install_utils/pe_utils.rb

```diff
@@ -425,12 +417,11 @@ def do_install hosts, opts = {}
               acceptable_exit_codes << 2 if opts[:type] == :upgrade
               setup_defaults_and_config_helper_on(host, master, acceptable_exit_codes)
             elsif host['platform'] =~ /windows/
-              on host, installer_cmd(host, opts)
+              msi_opts = { 'PUPPET_MASTER_SERVER' => master, 'PUPPET_AGENT_CERTNAME' => host }
+              opts = { :debug => host[:pe_debug] || opts[:pe_debug] }
+              msi_path = "#{host['working_dir']}\\#{host['dist']}.msi"
+              install_msi_on(host, msi_path, msi_opts, opts)
```

Comment:
@anodelman / @kevpl this is the fix for the previous acceptance failure on PE - when writing the batch file, it's preferential to fully qualify the MSI path for msiexec.exe, else you can run into working directory issues.

In the acceptance fail, we saw

```
r6xyhjiceqhydx4.delivery.puppetlabs.net (windows2008r2-64-2) 13:34:34$ cmd.exe /c "C:\Windows\TEMP\install-puppet-msi-2015-08-04_13.34.33.bat"

C:\cygwin64\home\Administrator>start /w msiexec.exe /i "puppet-enterprise-3.7.2-x64.msi" /qn /L*V C:\Windows\TEMP\install-puppet-2015-08-04_13.34.33.log PUPPET_MASTER_SERVER=fdhlj5dn9tqfqru.delivery.puppetlabs.net PUPPET_AGENT_CERTNAME=r6xyhjiceqhydx4.delivery.puppetlabs.net PUPPET_AGENT_STARTUP_MODE=Manual 

C:\cygwin64\home\Administrator>exit /B 1619 

r6xyhjiceqhydx4.delivery.puppetlabs.net (windows2008r2-64-2) executed in 0.39 seconds
Exited: 83
```

Re-running that same suite locally, after applying the change I see

```
adzftqfsuo3fi8b.delivery.puppetlabs.net (windows2008r2-64-2) 15:26:41$ cmd.exe /c "C:\Windows\TEMP\install-puppet-msi-2015-08-04_15.26.41.bat"

C:\cygwin64\home\Administrator>start /w msiexec.exe /i "C:\cygwin64\tmp\2015-08-04_15.19.50.iTbxtv\puppet-enterprise-3.7.2-x64.msi" /qn /L*V C:\Windows\TEMP\install-puppet-2015-08-04_15.26.41.log PUPPET_MASTER_SERVER=yohqe2hmop33snh.delivery.puppetlabs.net PUPPET_AGENT_CERTNAME=adzftqfsuo3fi8b.delivery.puppetlabs.net PUPPET_AGENT_STARTUP_MODE=Manual

C:\cygwin64\home\Administrator>exit /B 0

adzftqfsuo3fi8b.delivery.puppetlabs.net (windows2008r2-64-2) executed in 27.04 seconds
```